manim/mobject/three_d/three_dimensions.py

```diff
@@ -932,10 +952,12 @@ def __init__(
         thickness: float = 0.02,
         color: ParsableManimColor | None = None,
         resolution: int | Sequence[int] = 24,
-        **kwargs,
+        **kwargs: Any,
     ):
         self.thickness = thickness
-        self.resolution = (2, resolution) if isinstance(resolution, int) else resolution
+        self.resolution: Sequence[int] = (
+            (2, resolution) if isinstance(resolution, int) else resolution
+        )
```

Comment:
Is it mandatory to explicitly type `self.resolution` as `Sequence[int]` or, after my suggested changes, `tuple[int, int]`?